release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -81,6 +81,9 @@ With the replacement of Terraform, the following permissions are required if you
 
 For more information on required permissions, see xref:../installing/installing_azure/installing-azure-account.adoc#minimum-required-permissions-ipi-azure_installing-azure-account[Required Azure permissions for installer-provisioned infrastructure].
 ====
+==== Installing a cluster on {aws-full} using an existing IAM profile
```

Comment:
```suggestion
==== Installing a cluster on {aws-short} by using an existing IAM profile
```
I would suggest to consider using "AWS" in the heading and the full form in the paragraph. This also makes a heading smaller.